src/components/gui/gui.css

```diff
@@ -278,3 +278,15 @@ $fade-out-distance: 15px;
 .extension-button > div {
     margin-top: 0;
 }
+
+/* Alerts */
+
+.alerts-container {
+    width: 448px;
+    z-index: $z-index-alerts;
+    left: 0;
+    right: 0;
+    margin: auto;
+    position: absolute;
+    margin-top: 53px;
```

Comment:
@rschamp , I experimented with putting the alerts container inside of the flexWrapper so that this margin could be more like 5px vs. 53px, but then it seemed very strange to put the alerts container in such a wrapper that seemed to tightly bound to tabs/stage placement.  So, I left the alerts container in the main guiWrapper and made the margin bigger.